Целочисленная арифметика1/Целочисленная арифметика1/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace Целочисленная_арифметика1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            {
+                Console.WriteLine("Введите значение x");
+                var x = double.Parse(Console.ReadLine());
+
+                var y = F(x);
+
+                Console.WriteLine("f(x) = " + y);
+                Console.ReadKey();
+            }
+
+            static double F(double x)
+            {
+                return (1 / (Math.Pow(x, 2) + (1 / (Math.Pow(x, 2) + (1 / (Math.Pow(x, 2) + 1))))));
```

Comment:
Для вычисления квадрата лучше пользоваться умножением
```suggestion
                return (1 / (x * x + (1 / (x * x + (1 / (x * x + 1))))));
```